packages/contracts/src/utils/assertions.ts

```diff
@@ -0,0 +1,63 @@
+import * as chai from 'chai';
+import * as _ from 'lodash';
+
+import { constants } from './constants';
+
+const expect = chai.expect;
+
+function _expectEitherErrorAsync<T>(p: Promise<T>, error1: string, error2: string): PromiseLike<void> {
+    return expect(p)
+        .to.be.rejected()
+        .then(e => {
+            expect(e).to.satisfy(
+                (err: Error) => _.includes(err.message, error1) || _.includes(err.message, error2),
+                `expected promise to reject with error message that includes "${error1}" or "${error2}", but got: ` +
+                    `"${e.message}"\n`,
+            );
+        });
+}
+
+/**
+ * Rejects if the given Promise does not reject with an error indicating
+ * insufficient funds.
+ * @param p the Promise which is expected to reject
+ * @returns a new Promise which will reject if the conditions are not met and
+ * otherwise resolve with no value.
+ */
+export function expectInsufficientFundsAsync<T>(p: Promise<T>): PromiseLike<void> {
+    return _expectEitherErrorAsync(p, 'insufficient funds', "sender doesn't have enough funds");
```

Comment:
Let's consistently use single quotes. Surprised linter didn't catch this.

Reply:
Yeah, I believe that's what we do in the rest of the codebase. Only for consistency's sake.

Reply:
I tried changing it and prettier changed it back to what it is currently. I think that means I should leave it that way.